shared/protocol/src/main/java/io/pravega/shared/NameUtils.java

```diff
@@ -89,6 +89,11 @@
      */
     private static final String CHUNK_NAME_FORMAT_WITH_EPOCH_OFFSET = "%s.E-%d-O-%d.%s";
 
+    /**
+     * Format for name of read index block index entry.
```

Comment:
So what's the relationship between read index blocks and chunks? Up until this point I did not realize they were related to physical layout.

Reply:
Block index entry is a single KV pair. 
The key has name `<segment_name>.B-<block_start_offset>`. 
Value is very simple tuple for now. It contains **name of chunk** which has `block_start_offset`